exercises/isbn-verifier/canonical-data.json

```diff
@@ -149,6 +149,17 @@
             },
             "expected": false
         },
+        {
+            "uuid": "6ec400c2-6421-11ec-97c5-8c1645feb539",
```

Comment:
The validation is failing on this UUID. I think it's because you generated a version 1 UUID, but we expect a version 4 UUID.

Reply:
Good catch!

FYI to the OP that the appropriate UUID can be generated using Exercism's configlet tool.

Reply:
(Or on https://www.uuidgenerator.net/)

Reply:
Went with a third option by using the cli fom [OSSP UUID](https://github.com/exercism/problem-specifications/pull/1898): `uuid -v 4`,